### DISCUSSION TITLE
Parameters not transfered to postal in API request

### DISCUSSION THREAD

**Discussion by thomaswennersten** [postalserver/postal#2692] [Help with using Postal]
I have made a request to the postal API that looks like below, but it do not seem to work since the defined parameters not seem to be transferred correct to postal.. I have done according to the documentation. Result in the bottom pic.

![image](https://github.com/postalserver/postal/assets/89129225/612d7f7f-e0e5-4191-a7dc-7eefe3e1da8a)

{
  "status": "error",
  "time": 0,
  "data": {
    "code": "NoRecipients",
    "message": "There are no recipients defined to receive this message"
  }
}

**Comment by fernandoSynetic** [comment]
It is possible that you have empty "TO" and "CC" is not an array.

**Reply by willpower232** [reply]
Yeah not familiar with that program but definitely needs to be an array https://apiv1.postalserver.io/controllers/send/message.html

Postal is really good at ignoring invalid inputs :sweat_smile:

**Reply by thomaswennersten** [reply]
I have changed the to parameter to an array with this syntax: ["thomas@gmail.com","thomas@skrotruk.se"] in the request but still no luck.

SInce i use a lowcode tool to make the request and not my own code I would like to se in Postal what it receives. Is there a way to check a log in Potsal to see? I have full server access.

**Reply by willpower232** [reply]
Unfortunately not, you could post to something like pastebin I guess.

Which "lowcode tool" are you using?

**Reply by thomaswennersten** [reply]
I use something called NocoBase, it is realy god, but the effort of creating http requests for notifications seems to demand some effort...

I have set up mailhog and hope that can catch my request so I can view it

**Reply by thomaswennersten** [reply]
@willpower232 could you maybe share an example of the syntax of a POST request to postal API looks like?

I have now found a way to actualy verify what is sent from my lowcode plattform and it looks lite below.

![image](https://github.com/postalserver/postal/assets/89129225/ef726a72-8c98-40ae-9581-d4ffeb566bcf)

**Reply by willpower232** [reply]
Your screenshot suggests that you are sending an empty body with all the fields actually in the query string?

![image](https://github.com/postalserver/postal/assets/1619102/9d4a0cf5-3f1b-449b-ac27-aacf3c4d6c4d)

**Reply by thomaswennersten** [reply]
Yes, that was the case. I have now moved the parameters into the body and now it works.

Thank´s for response!
